### PR TITLE
Stop incrementing LMP after appointment date has passed

### PR DESCRIPTION
**Pull request by yf6**
Also fixes tests for LastMenstrualPeriodHelper.

Fixes https://github.com/colinxfleming/dcaf_case_management/issues/931

@colinxfleming

### CODE REVIEW

File: test/helpers/last_menstrual_period_helper_test.rb [hunk 1]
```diff
@@ -3,94 +3,101 @@
...
+      assert_nil @pregnancy.send(:last_menstrual_period_now)
+    end
+
+    it 'should be equivalent to LMP on date - Time.zone.today' do
+      assert_equal  @pregnancy.send(:last_menstrual_period_now),
+                    @pregnancy.send(:last_menstrual_period_on_date,
+                                    Time.zone.today)
+    end
+
+    it 'should be LMP on date of appointment if appointment is in the past' do
```
Comment (yf6) [inline test/helpers/last_menstrual_period_helper_test.rb]: this is the only new test. everything else was there already

File: test/helpers/last_menstrual_period_helper_test.rb [hunk 2]
```diff
@@ -3,94 +3,101 @@
...
-                                appointment_date: 2.days.from_now
-
     @pregnancy = create :pregnancy,
                         last_menstrual_period_weeks: 9,
                         last_menstrual_period_days: 2,
                         created_by: @user
+    @patient = create :patient, created_by: @user,
+                                initial_call_date: 2.days.ago,
+                                appointment_date: 2.days.from_now,
+                                pregnancy: @pregnancy
```
Comment (yf6) [inline test/helpers/last_menstrual_period_helper_test.rb]: this is what i had to do to fix the existing tests, patient needs to include pregnancy so the patient info can be pulled into the tests

**Comment by lomky** [comment]
Starting review on this PR!

**Comment by yf6** [comment]
<3

**Pull request merged**